citizens-advice-style.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(spec)/}) }
   end
```

Comment:
The spec directory doesn't exist?  Is this just a default thing that gets added to a gemspec file?

Reply:
Yep. Just removing excess crud.